src/utilities/index.ts

```diff
@@ -1,29 +1,54 @@
 import Link from 'gatsby-link'
 
-export function checkBreakpoint(breakpoint) {
+export function checkBreakpoint(breakpoint: number) {
   if (typeof window !== `undefined`) {
     return window.matchMedia(`(min-width: ${breakpoint}px)`).matches
   }
 }
 
-export function externalUrl(url) {
+export function externalUrl(url: string) {
   const external = /^(http|https|ftp)/
   return external.test(url)
 }
 
-export function contactUrl(url) {
+export function contactUrl(url: string) {
   const contact = /^(mailto:|tel:)/
   return contact.test(url)
 }
 
-export function handleSlug(slug) {
-  const hasLeadingSlash = /^[\/]/
+export function handleSlug(slug: string) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'handleSlug' to have a typedef](https://app.codacy.com/app/PrideInLondon/pride-london-web/pullRequest?prid=4553972)